tests/coord.test.js

```diff
@@ -51,4 +52,29 @@ test('validate Coord: default values', () => {
     expect(defCoord.validate()).toBe(false);
     expect(defXCoord.validate()).toBe(false);
     expect(defYCoord.validate()).toBe(false);
-})
\ No newline at end of file
+});
+
+test('verify toString output', () => {
+    let c = new Coord();
+
+    for (let i = 0; i < 100; i++) {
+        c.setLocation(i, i);
+        expect(c.toString()).toBe('(' + i + ', ' + i + ')');
+    }
+});
```

Comment:
This seems kinda excessive to me. Why not just test an invalid value, a default value, and a regular value?

Reply:
It's been simplified